cmd/cloud-controller-manager/BUILD

```diff
@@ -1,26 +1,48 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_binary",
+    "go_library",
+)
+load("//vendor/k8s.io/component-base/version:def.bzl", "version_x_defs")
+
+go_binary(
+    name = "cloud-controller-manager",
```

Comment:
Why?

Reply:
It was picked up from https://github.com/kubernetes/kubernetes/blob/release-1.18/cmd/cloud-controller-manager/BUILD#L11